components/SizerBase/src/SizerBase.Properties.cs

```diff
@@ -90,10 +90,28 @@ public Orientation Orientation
     public static readonly DependencyProperty OrientationProperty =
         DependencyProperty.Register(nameof(Orientation), typeof(Orientation), typeof(SizerBase), new PropertyMetadata(Orientation.Vertical, OnOrientationPropertyChanged));
 
+    /// <summary>
+    /// Gets or sets if the indicator is visible. If not visible, only the background and cursor will be shown on MouseOver or Pressed states.
+    /// </summary>
+    public bool IsIndicatorVisible
+    {
+        get { return (bool)GetValue(IsIndicatorVisibleProperty); }
+        set { SetValue(IsIndicatorVisibleProperty, value); }
+    }
+
+    /// <summary>
+    /// Identifies the <see cref="IsIndicatorVisible"/> dependency property.
+    /// </summary>
+    public static readonly DependencyProperty IsIndicatorVisibleProperty =
+        DependencyProperty.Register(nameof(IsIndicatorVisible), typeof(bool), typeof(SizerBase), new PropertyMetadata(true, OnIsIndicatorVisiblePropertyChanged));
+
+
     private static void OnOrientationPropertyChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
     {
         if (d is SizerBase gripper)
         {
+            VisualStateManager.GoToState(gripper, gripper.Orientation == Orientation.Vertical ? "Vertical" : "Horizontal", true);
```

Comment:
Should we add the [`TemplateVisualState`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.templatevisualstateattribute) attributes (that for some reason most UWP controls don't have)? Then we can create a const for the string to use there and here in case they need to be reused elsewhere?

Reply:
Updated 👍